sdk/batch/Microsoft.Azure.Management.Batch/tests/ScenarioTests/PoolTests.ScenarioTests.cs

```diff
@@ -118,8 +119,8 @@ public async Task BatchPoolEndToEndAsync()
                     // Verify get operation
                     var pool = await this.BatchManagementClient.Pool.GetAsync(resourceGroupName, batchAccountName, iaasPoolName);
                     Assert.Equal("STANDARD_A1", pool.VmSize);
-                    Assert.Null(pool.DisplayName);
-                    Assert.Equal(AllocationState.Steady, pool.AllocationState);
+                    Assert.Equal(displayName, pool.DisplayName);
+                    Assert.Equal(AllocationState.Resizing, pool.AllocationState);
```

Comment:
This has always been the expected behavior, however, the transition to `Steady` from `Resizing` is now more latent, causing the faulty assert to fail consistently.